server.go

```diff
@@ -462,7 +462,7 @@ func (s *Server) Gen(def *Def, r io.Reader, w io.Writer) error {
 					return err
 				}
 
-				fmt.Fprintln(w, alias)
+				fmt.Fprintln(w, "0", alias)
```

Comment:
😅 Still wrong. This is the `Gen` method to this should be 1 since this is where the alias is being created. On line 425, it should be 0 since it was found.

Line 493 is correct in `Get`, but you are missing line 502, that should print 0 since it was not found.